meson.build

```diff
@@ -1445,106 +1446,105 @@ endif
+    endif
 
-if macos_sdk_version != '0.0'
-    message('Detected macOS SDK: ' + macos_sdk_version)
-    add_languages('objc')
-    objc_link_flags = ['-isysroot', macos_sdk_path, '-L/usr/lib', '-L/usr/local/lib']
-    add_project_link_arguments(objc_link_flags, language: ['c', 'objc'])
-endif
+    if macos_sdk_version != '0.0'
+        message('Detected macOS SDK: ' + macos_sdk_version)
+        add_languages('objc')
+        objc_link_flags = ['-isysroot', macos_sdk_path, '-L/usr/lib', '-L/usr/local/lib']
+        add_project_link_arguments(objc_link_flags, language: ['c', 'objc'])
+    endif
 
-macos_10_11_features = get_option('macos-10-11-features').require(
-    macos_sdk_version.version_compare('>=10.11'),
-    error_message: 'a suitable macos sdk version could not be found!',
-)
+    macos_10_11_features = get_option('macos-10-11-features').require(
+        macos_sdk_version.version_compare('>=10.11'),
+        error_message: 'a suitable macos sdk version could not be found!',
+    )
 
-macos_10_12_2_features = get_option('macos-10-12-2-features').require(
-    macos_sdk_version.version_compare('>=10.12.2'),
-    error_message: 'a suitable macos sdk version could not be found!',
-)
+    macos_10_12_2_features = get_option('macos-10-12-2-features').require(
+        macos_sdk_version.version_compare('>=10.12.2'),
+        error_message: 'a suitable macos sdk version could not be found!',
+    )
 
-macos_10_14_features = get_option('macos-10-14-features').require(
-    macos_sdk_version.version_compare('>=10.14'),
-    error_message: 'a suitable macos sdk version could not be found!',
-)
+    macos_10_14_features = get_option('macos-10-14-features').require(
+        macos_sdk_version.version_compare('>=10.14'),
+        error_message: 'a suitable macos sdk version could not be found!',
+    )
 
-xcrun = find_program('xcrun', required: get_option('swift-build'))
-swift_ver = '0.0'
-if xcrun.found()
-    swift_prog = find_program(run_command(xcrun, '-find', 'swift', check: true).stdout().strip())
-    swift_ver_string = run_command(swift_prog, '-version', check: true).stdout()
-    verRe = '''
-#!/usr/bin/env python3
-import re
-import sys
-verRe = re.compile("(?i)version\s?([\d.]+)")
-swift_ver = verRe.search(sys.argv[1]).group(1)
-sys.stdout.write(swift_ver)
-'''
-    swift_ver = run_command(python, '-c', verRe, swift_ver_string, check: true).stdout()
-    message('Detected Swift version: ' + swift_ver)
-endif
+    xcrun = find_program('xcrun', required: get_option('swift-build'))
+    swift_ver = '0.0'
+    if xcrun.found()
+        swift_prog = find_program(run_command(xcrun, '-find', 'swift', check: true).stdout().strip())
+        swift_ver_string = run_command(swift_prog, '-version', check: true).stdout()
+        verRe = '''
+    #!/usr/bin/env python3
+    import re
+    import sys
+    verRe = re.compile("(?i)version\s?([\d.]+)")
+    swift_ver = verRe.search(sys.argv[1]).group(1)
+    sys.stdout.write(swift_ver)
+    '''
```

Comment:
this probably fails because of the indentation now but you could also directly shorten this to
```py
verRe = '''import sys,re;sys.stdout.write(re.search("(?i)version\s?([\d.]+)", sys.argv[1]).group(1))'''
```